main.tex

```diff
@@ -117,21 +117,21 @@ \subsection{Input Registration}
 % - single pedersen multicommitment for amount and serial or two separate group attributes?
 % - if separate, extra generator + randomness for unconditional hiding of serial number even after revealing serial?
 
-The coordinator verifies the proofs, and issues $k$ MACs (message authentication codes) on the requested attributes, along with a proof of knowledge of the secret key as described in \textit{Credential Issuance} protocol of \cite{chase2019signal}.
+The coordinator verifies the proofs, and issues $k$ MACs on the requested attributes, along with a proof of knowledge of the secret key as described in \textit{Credential Issuance} protocol of \cite{chase2019signal}.
```

Comment:
Why?

Reply:
@seresistvanandras is adding definitions of these so it's now somewhat redundant with that addition

Reply:
MAC is already written out long in line 92
https://github.com/zkSNACKs/WabiSabi/pull/24/files#diff-a45f2757f94fd2337ebf7ddd018c11a2L92

I think it is enough to write it out long once, and go with the short afterwards